app/controllers/calendar_controller.rb

```diff
@@ -74,10 +74,16 @@ def simple_404
 
   def handle_bank_holiday_ics_calendars
     if scope == "bank-holidays"
-      I18n.backend.send(:init_translations) unless I18n.backend.initialized?
```

Comment:
I'd like to investigate what all this code intends to do and make it better, instead of doing the `I18n.backends` juggling. It looks like it's using all kinds of private APIs now.